doc/devel/contributing.rst

```diff
@@ -102,34 +112,39 @@ A brief overview is:
 
 3. Clone this copy to your local disk::
 
-      $ git clone https://github.com/YourLogin/matplotlib.git
+      git clone https://github.com/<YOUR GITHUB USERNAME>/matplotlib.git
+
+4. Enter the directory and install the local version of Matplotlib. For more
+   detailed instructions on doing this see ref`<installing_for_devs>`::
 
-4. Create a branch to hold your changes::
+      cd matplotlib
+      pip install -ve . 
```

Comment:
```suggestion
      pip install -ve . 
    
    NOTE: `-e` flag would make your installation 'editable', such that your work is always
    in sync as the installed version. For more details see [ref](https://pip.pypa.io/en/stable/reference/pip_install/#install-editable).
```

Reply:
I think this resolved by entirely deferring this to the standalone dev install page